roles/elasticsearch/tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: Install elasticsearch
+  apt:
+    name: elasticsearch
+    state: present
+
+- name: Configure Elasticsearch.
+  template:
+    src: elasticsearch.yml.j2
```

Comment:
Nit: We don't really have .j2 on our templates. Also, we keep them in templates/ with similar path that they'll be deployed to.